doc/apiary.apib

```diff
@@ -0,0 +1,754 @@
+                            "id": 12
+                        }
+                    ],
+                    "id": 3,
+                    "label": "SensorModel"
+                }
+            }
+
++ Response 400 (application/json)
+
+            {
+                "errors": {
+                    "label": [
+                        "Missing data for required field."
+                    ]
+                },
+                "message": "failed to parse input"
+            }
+
+
+### Get the current list of templates [GET]
+Get the full list of templates with all their associated attributes.
+
++ Request
+      + Headers
+
+            Authorization: Bearer JWT
+
++ Response 200 (application/json)
+
+            {
+                "pagination": {
+                    "has_next": false,
+                    "next_page": null,
+                    "page": 1,
+                    "total": 1
+                },
+                "templates": [
+                    {
+                        "attrs": [
+                            {
+                                "created": "2017-12-20T18:42:21.089367+00:00",
+                                "id": 13,
+                                "label": "temperature",
+                                "template_id": "4",
+                                "type": "dynamic",
+                                "value_type": "float"
+                            },
+                            {
+                                "created": "2017-12-20T18:42:21.090601+00:00",
+                                "id": 14,
+                                "label": "pressure",
+                                "template_id": "4",
+                                "type": "dynamic",
+                                "value_type": "float"
+                            },
+                            {
+                                "created": "2017-12-20T18:42:21.091596+00:00",
+                                "id": 15,
+                                "label": "led",
+                                "template_id": "4",
+                                "type": "dynamic",
+                                "value_type": "bool"
+                            },
+                            {
+                                "created": "2017-12-20T18:42:21.092641+00:00",
+                                "id": 16,
+                                "label": "fan",
+                                "template_id": "4",
+                                "type": "dynamic",
+                                "value_type": "bool"
+                            }
+                        ],
+                        "created": "2017-12-20T18:42:21.087782+00:00",
+                        "id": 4,
+                        "label": "SensorModel"
+                    }
+                ]
+            }
+
+
+## Template info [/template/{id}]
+Access a specific template
++ parameters
+    + id: `83af` (required, string) - The template ID
```

Comment:
This example id does not match the id format that was returned on the previous call, with the previous being an integer and this being part of a hash

Reply:
I'll fix that.